### PR TITLE
add chisq test functions

### PR DESCRIPTION
**Pull request by double-y**
### Description
* Add functions for chi-square test
* export some utility functions

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [x] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/test_wrapper.R [hunk 1]
```diff
@@ -46,7 +46,7 @@ do_t.test <- function(df, value, key=NULL, ...){
               estimate1 = "mean1",
               estimate2 = "mean2",
               statistic = "t.value",
-              parameter = "degrees_of_freedom",
+              parameter = "digrees_of_freedom",
```
Comment (hideaki) [inline R/test_wrapper.R]: typo: "digrees"

File: R/util.R [hunk 2]
```diff
@@ -655,3 +668,38 @@ get_confint <- function(val, se, conf_int = 0.95) {
...
+#' NSE version of pivot_
+#' @export
+pivot <- function(data, formula, value = NULL, ...) {
+  value_col <- col_name(substitute(value))
+  pivot_(data, formula = formula, value_col = value_col, ...)
+}
+
+#' pivot columns based on formula
+#' @param data Data frame to pivot
+#' @param formula lhs is composed of columns for rows and rhs is for cols
```
Comment (hideaki) [inline R/util.R]: Please explain more about how to use this formula.

File: R/test_wrapper.R (hunk 1 above)
Comment (double-y) [inline R/test_wrapper.R]: Oh, right. Fixed it

File: R/util.R (hunk 2 above)
Comment (double-y) [inline R/util.R]: Added explanation

**Pull request merged**